buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring_1_26_0.groovy

```diff
@@ -23,53 +23,54 @@ package org.apache.beam.gradle
  */
 class GrpcVendoring_1_26_0 {
 
-  static def guava_version = "26.0-jre"
```

Comment:
This one and the next one are only sorting stuff to make it more easy to find the refs.